bin/auditlog_functions.js

```diff
@@ -24,8 +25,8 @@ module.exports.dispatcher = function (args) {
             printModuleHelp();
             break;
         default:
-            util.error("Unknown operatation");
-            break;
+            util.error("Unknown operation");
+            process.exit(100);
```

Comment:
I think this error state is neither a testengine failure nor a testjob error. We kinda repurpose the 100-code a bit. I don't think it is super important, but I think someone else might. Could be worth asking about it. Especially since we say that "unknown operation" is a '1' later.

Reply:
Yes, this is not related to testjob and testengine. May be it should not be exited with error code. Instead just give the message as before